app/Controllers/StaticPagesController.js

```diff
@@ -0,0 +1,47 @@
+const catchAsync = require('../utils/catchAsync');
+const path = require('path');
+const ApiError = require('../utils/ApiError.js');
+const httpStatus = require('http-status');
+
+const renderHandler = catchAsync(async (res, page) => {
+  const options = { root: path.join(__dirname, '../../') };
+
+  // check if page should be rendered from a static file or should be redirected to another one
+  if (page.type === 'file') {
+    res.sendFile(page.location, options, (err) => {
```

Comment:
shouldnt we use a template engine here? Or is that something for later PRs? By using a template engine, we are able to let the template engine decide which page should be included.

Reply:
I think this is more like a task for a later PR. This if query is just to determine whether we should use ``sendFile`` or ``redirect``